mleap-spark/build.sbt

```diff
@@ -2,3 +2,12 @@ import ml.combust.mleap.{Dependencies, Common}
 
 Common.defaultMleapSettings
 Dependencies.spark
+
+val printDependencyClasspath: TaskKey[Unit] = taskKey[Unit]("Prints locations of the dependencies")
+
+printDependencyClasspath := {
+  val cp = (dependencyClasspath in Compile).value
+  // TODO write to file so that this can be read by pyspark tests? or how should this work, maybe pyspark tests should
```

Comment:
I'm not sure what you are trying to do here so I am not sure how to answer question 3) in this PR

Reply:
@lucagiovagnoli thanks, what I actually want to achieve is importing the contents of mleap jar (and dependencies) in the SparkSession of pyspark tests, but in a quick & easy-to-develop way. When using a published mleap from maven, it's simple:

```python
SparkSession.builder \
    .config("spark.jars.packages", "ml.combust.mleap:mleap-spark-extension_2.11:0.14.0")
```

However, in development mode I want to use the local version of mleap code.

I found that it's possible to make it work by listing the paths to all classes/ folders of locally built mleap & dependency jars. However, this is kinda manual. I was wondering if sbt would provide some easier way to export the required runtime classpath (when sbt is for example running tests locally) for the `mleap-spark-extension` module.

Another way to do this could be to install mleap to local maven repo with with some snapshot version and use it with `"spark.jars.packages"` (but obviously that's a bit slow for testing new changes, and not too clean, because it's writing to your local maven repo).

Reply:
Ok, here's a clean way to get the classpath on command line:

`sbt "export mleap-spark-extension/runtime:fullClasspath"`

However it prints some additional log lines first.. Maybe I will still look into a programmatic way.